### PR TITLE
RootFileReader: Read TH1, improve TH2 reading + tests

### PR DESCRIPTION
**Pull request by AndreasAlbert**
* Added reader for TH1
* TH2 reader now also returns bin errors
* Tests for both

### CODE REVIEW

**Comment by AndreasAlbert** [comment]
Closes #9

**Comment by AndreasAlbert** [comment]
@clelange Can we merge this, please?

File: hepdata_lib/__init__.py [hunk 1]
```diff
@@ -666,10 +679,42 @@ def get_hist_2d_points(hist):
...
+    points = {}
+    for key in ["x", "y", "x_edges", "dy"]:
+        points[key] = []
+
+    for x_bin in range(1, hist.GetNbinsX() + 1):
+        x_val = hist.GetXaxis().GetBinCenter(x_bin)
+        width_x = hist.GetXaxis().GetBinWidth(x_bin)
+
+        y_val = hist.GetBinContent(x_bin)
+        dy_val = hist.GetBinError(x_bin)
```
Comment (clelange) [inline hepdata_lib/__init__.py]: Shouldn't we use asymmetric errors here, i.e. `GetBinErrorLow()` and `GetBinErrorUp()`?

File: tests/test_rootfilereader.py [hunk 2]
```diff
@@ -3,11 +3,13 @@
 """Test RootFileReader."""
 import random
 from unittest import TestCase
-import test_utilities
+from test_utilities import *
```
Comment (clelange) [inline tests/test_rootfilereader.py]: I'd prefer to avoid `import *`, is it really necessary?

File: hepdata_lib/__init__.py (hunk 1 above)
Comment (AndreasAlbert) [reply hepdata_lib/__init__.py]: I didn't realize this existed. Added it now.

File: tests/test_rootfilereader.py (hunk 2 above)
Comment (AndreasAlbert) [reply tests/test_rootfilereader.py]: You are right. Fixed now.

**Pull request merged**